ci-operator/step-registry/ipi/conf/nutanix/realtime/ipi-conf-nutanix-realtime-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Create node MachineConfig files and set kernelType to realtime
+cat > "${SHARED_DIR}"/manifest_99-worker-kerneltype.yaml <<EOF
```

Comment:
This new step does not need `RT_ENABLED` env, shall we integrate the step into ipi-conf-nutanix controlled by `RT_ENABLED`, just like gpc/ibmcloud platform?